webapp/channels/src/components/suggestion/command_provider/command_provider.tsx

```diff
@@ -140,7 +140,11 @@ export default class CommandProvider extends Provider {
     }
 
     handleCompleteWord(term: string, pretext: string, callback: (s: string) => void) {
-        callback(term + ' ');
+        let newValue = term + ' ';
+        if (term === '/join') {
```

Comment:
would it be worthwhile to convert this logic into something more generic, and also cater for other commands that have the same behaviour? e.g. `/mute`

Reply:
I think you could try to set a dictionary with command and the symbol to follow, then find the term in the dictionary and use the symbol to create the new value. Perhaps you can start with join and mute and see how it works. But feel free to open a new PR for that. This one is ok as it is, given that is solves what was initially requested.